syntax = "proto3";

message HelloWorld {
    string text = 1;
}

message ResetCause {
    int32 supported_causes = 1;
    int32 reset_cause = 2;
}